module/skills/KickScript/src/KickScript.cpp

```diff
@@ -0,0 +1,52 @@
+#include "KickScript.hpp"
+
+#include "extension/Behaviour.hpp"
+#include "extension/Configuration.hpp"
+#include "extension/behaviour/Script.hpp"
+
+#include "message/actuation/Limbs.hpp"
+#include "message/skills/Kick.hpp"
+
+#include "utility/input/LimbID.hpp"
+
+namespace module::skills {
```

Comment:
I think this should be `skill` not `skills` simply because every other level of behaviour is also singular